src/main/scala/io/viash/platforms/nextflow/NextflowHelper.scala

```diff
@@ -184,11 +184,16 @@ object NextflowHelper {
     * 
     *   - For local dependencies (i.e. the dependency's source code is defined in the same project as the current repository):
     *     ```
-    *     include { my_dep as my_alias } from "$projectDir/../../../target/nextflow/my_namespace/my_dep/main.nf"
+    *     include { my_dep } from "$projectDir/../../../target/nextflow/my_namespace/my_dep/main.nf"
     *     ```
     *   - For remote dependencies (i.e. the dependency is fetched from a different project -- either a local folder or a remote repository):
     *     ```
-    *     include { my_dep as my_alias } from "$rootDir/dependencies/my_namespace/my_dep/main.nf"
+    *     include { my_dep } from "$rootDir/dependencies/my_namespace/my_dep/main.nf"
+    *     ```
+    *   - When an alias is defined:
+    *     ```
+    *     include { my_dep as my_alias_viashalias } from "..."
```

Comment:
To be consistent with the example above?

```suggestion
    *     include { my_dep as my_alias_viashalias } from "$rootDir/dependencies/my_namespace/my_dep/main.nf"
```

Reply:
I used `"..."` because we don't know whether it's a local or a remote dependency given the lack of context :relaxed: